docs/completeExamples/completeExamples.xml

```diff
@@ -0,0 +1,485 @@
+            <Description>User has queried database using specified constraints</Description>
+            <Search>
+                <Query>
+                    <Raw>type=r, size=large, colour=white</Raw>
+                </Query>
+                <TotalResults>2</TotalResults>
+                <Results>
+                    <Object>
+                        <Type>Rock</Type>
+                        <Id>78121</Id>
+                        <Name>Surpisingly Heavy Chunk</Name>
+                    </Object>
+                    <Object>
+                        <Type>Rock</Type>
+                        <Id>11418</Id>
+                        <Name>Possible Gold Ore</Name>
+                    </Object>
+                </Results>
+            </Search>
+        </EventDetail>
+    </Event>
+    <Event>
+        <EventTime>
+            <TimeCreated>2017-01-02T03:04:05.678Z</TimeCreated>
+        </EventTime>
+        <EventSource>
+            <System>
+                <Name>Space Desk</Name>
+                <Environment>LunaDomain</Environment>
+                <Organisation>ACMECoolResearch</Organisation>
+            </System>
+            <Generator>RMMaster2000 RM Control</Generator>
+            <Device>
+                <HostName>rmmaster.luna.mydomain.org</HostName>
+            </Device>
+            <Client>
+                <Name>LUNA/LUNADESK35</Name>
+                <HostName>lunadesk35.lunadesk.luna.mydomain.org</HostName>
+            </Client>
+            <User>
+                <Id>jc101</Id>
+                <Domain>ACMECoolResearch/Users</Domain>
+                <UserDetails>
+                    <Id>jc101</Id>
+                    <StaffNumber>101</StaffNumber>
+                    <Surname>Coder</Surname>
+                    <Initials>JD</Initials>
+                    <Title>Mr</Title>
+                    <KnownAs>Jolly</KnownAs>
+                    <PersonType>Employee</PersonType>
+                    <Group>Technology</Group>
+                    <Unit>Geology</Unit>
+                    <Position>T.SD/42</Position>
+                    <Role>Geologist</Role>
+                    <GradeOfPost>E.5</GradeOfPost>
+                    <EmploymentType>Employee</EmploymentType>
+                    <EmploymentStatus>Active</EmploymentStatus>
+                    <Nationality>GB</Nationality>
+                    <Location>Moon Base Alpha</Location>
+                    <RoomNumber>4/31</RoomNumber>
+                    <Phone>11223</Phone>
+                    <SupervisorStaffNumber>421</SupervisorStaffNumber>
+                    <Organisation>ACMECoolResearch</Organisation>
+                    <HostOrganisation>MoonCo</HostOrganisation>
+                </UserDetails>
+            </User>
+            <Interactive>true</Interactive>
+        </EventSource>
+        <EventDetail>
+            <TypeId>ReadRM</TypeId>
+            <Description>User has read a file from removable media</Description>
+            <Import>
```

Comment:
With import and export events I think it is worth capturing both ends, e.g. you may be importing a file into a system, so the source is a file and its path and the dest is say a record/document within the application that may have been given some form of identifier.